var/spack/repos/builtin/packages/boost/package.py

```diff
@@ -581,7 +583,7 @@ def determine_b2_options(self, spec, options):
 
         options.extend(["link=%s" % ",".join(link_types), "--layout=%s" % layout])
 
-        if not spec.satisfies("@:1.75 %intel"):
+        if not (spec.satisfies("@:1.75 %intel") or spec.satisfies("platform=windows")):
```

Comment:
I'm assuming the idea is you don't want to set `toolset=` if we're on windows, in that case, this `if` would need to be:

`if not (spec.satisfies("@:1.75 %intel") and not spec.satisfies("platform=windows")):`

Reply:
I misread what you originally had, and I should have been OK with it. Thanks for making the change (either formulation is fine).